book/_quarto.yml

```diff
@@ -44,7 +44,7 @@ knitr:
     echo: true
     fig.width: 15
 
-editor: visual
```

Comment:
where is this coming from?

Reply:
Oh sorry, I'll revert that. It just stops the qmd docs from automatically opening in visual editor mode - I prefer the other mode for development so use this on my local.